src/wallet/api/wallet2_api.h

```diff
@@ -408,6 +408,12 @@ struct Wallet
     */
     virtual std::string publicSpendKey() const = 0;
 
+    /*!
+     * \brief publicSignerKey  - returns public signer key
+     * \return                 - public signer key for multisig wallet or spend public for classic wallet
```

Comment:
>  or spend public for classic wallet

Wait, this description doesn't agree with the implementation and should be dropped.

How about renaming the function to something like `publicMultisigSignerKey()` to emphasize that it's specific to multisig?

Reply:
no problem!

Reply:
Renamed method and updated descritption